app/subscriber/src/features/home/Home.tsx

```diff
@@ -53,15 +55,32 @@ export const Home: React.FC = () => {
     [findContentWithElasticsearch],
   );
 
+  const featuredItemsName = React.useMemo(() => {
+    const value = appSettings?.find((s) => s.name === Settings.FeaturedItemsName)?.value;
+    if (!value && !!appSettings?.length)
```

Comment:
just in case its not set

Reply:
Don't use a featured name in Settings.  Use the key so that in the future the admin can change the name and everything works.  Just like it was Homepage and now is Featured Story.

Reply:
Just need some clarification, `Settings.FeaturedItemsName` is the key for "Featured Story" and it can be configured in Settings if the name changes again

Reply:
![image](https://github.com/bcgov/tno/assets/15724124/dba4ab0b-7565-48dc-a757-71d4a9534f65)

Reply:
Let me know if I am misunderstanding @Fosol

Reply:
The Settings should almost always point to primary keys.  An admin can one day change the name of "Featured Stories" to "Today's Featured Stories".  Just like they changed it from "Homepage" to "Featured Stories".  The setting would keep working if it used the primary key.

With your implementation it would break.

Reply:
Ah I was avoiding using `id` because of some overthought that the action could change resulting in a different `id`. Will fix

Reply:
A primary key cannot change.  If they deleted the action then the filter would stop working.  Which would be expected behaviour.  They would need to then go to the settings and choose a new action to filter on.